media/js/firefox/whatsnew/whatsnew-61.js

```diff
@@ -0,0 +1,148 @@
+        Display the Send to Device widget, configured for Focus
+    ii. Send to Device widget is *not* available in the user's locale:
+        Display a QR code for Focus
+*/
+
+(function(Mozilla, $) {
+    'use strict';
+
+    var client = Mozilla.Client;
+    var mainContent = document.querySelector('.main-content');
+    var $strings = $('#strings');
+
+    function showFxa() {
+        // Show the content
+        mainContent.classList.add('show-fxa');
+
+        // Set the title
+        document.title = $strings.data('fxaccount-title');
+
+        // initialize the FxA iframe
+        Mozilla.Client.getFirefoxDetails(function(data) {
+            Mozilla.FxaIframe.init({
+                distribution: data.distribution,
+                gaEventName: 'whatsnew-fxa'
+            });
+        });
+    }
+
+    function showFirefoxMobile() {
+        // Show the content
+        mainContent.classList.add('show-fx-mobile');
+
+        // Set the title
+        document.title = $strings.data('fxmobile-title');
+
+        var form = new Mozilla.SendYourself('send-firefox');
+        form.init();
+    }
+
+
+    function showFocus(countryCode) {
+        var logoFx = document.getElementById('logo-fx');
+        var logoFocus = document.getElementById('logo-focus');
+
+        // Show the content
+        mainContent.classList.add('show-focus');
+
+        // Set the title
+        document.title = $strings.data('fxfocus-title');
+
+        // swap out Firefox logo for Focus logo
+        logoFx.classList.replace('showing', 'hiding');
+
+        setTimeout(function() {
+            logoFocus.classList.replace('hiding', 'showing');
+        }, 150);
+
+        var form = new Mozilla.SendYourself('send-focus');
+        form.init(countryCode);
+    }
+
+
+    function showKlar(countryCode) {
+        var logoFx = document.getElementById('logo-fx');
+        var logoFocus = document.getElementById('logo-focus');
+
+        // Show the content
+        mainContent.classList.add('show-klar');
+
+        // Set the title
+        document.title = $strings.data('fxfocus-title');
+
+        // swap out Firefox logo for Focus logo
+        logoFx.classList.replace('showing', 'hiding');
+
+        setTimeout(function() {
+            logoFocus.classList.replace('hiding', 'showing');
+        }, 150);
+
+        var form = new Mozilla.SendYourself('send-klar');
+        form.init(countryCode);
+    }
+
+
+    // bug 1419573 - only show "Your Firefox is up to date" if it's the latest version.
+    if (client.isFirefoxDesktop) {
+        client.getFirefoxDetails(function(data) {
+            if (data.isUpToDate) {
+                document.querySelector('.main-header').classList.add('show-up-to-date-message');
+            }
+        });
+    }
+
+    client.getFxaDetails(function(details) {
+        // TODO: verify these are the correct 2-letter country codes
+        var klarCountryCodes = ['at', 'ch', 'de'];
```

Comment:
We sure these are the proper country codes? (The internet says yes.) If so, remove the `// TODO`.